sdk/python/feast/type_map.py

```diff
@@ -548,9 +547,11 @@ def snowflake_python_type_to_feast_value_type(
     return type_map[snowflake_python_type_as_str.lower()]
 
 
-def pa_to_redshift_value_type(pa_type: pyarrow.DataType) -> str:
+def pa_to_redshift_value_type(pa_type: "pyarrow.DataType") -> str:
```

Comment:
@achals @adchia one blocker for this endeavor is this pyarrow library in type_map.py. I want to reuse the code here (and not have to maintain a separate file) in my snowflake udfs but snowflake doesnt support the pyarrow package as of today.

Given pyarrow is only required for certain infra configs, I moved the import into the function and made the type hint a string hint -- which I am not sure how feast feels about ... looking for advice here

Reply:
I think moving the import inside the func and using a stringified type hint is fine.

If you want, you can also do a 
```
if typing.TYPE_CHECKING:
    import pyarrow
```

in the imports section.

Reply:
Snowflake should be able to accept the origin code in 2 weeks without having to do anything wonky with pyarrow